docs/src/pages/_app.page.tsx

```diff
@@ -35,6 +35,10 @@ function MyApp({ Component, pageProps }) {
       <Head>
         <title>Amplify UI</title>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta
+          name="description"
+          content="Amplify UI is an open-source design system with cloud-connected components and primitives that simplify building accessible, performant, and beautiful applications in React, Angular, Vue, and Flutter (more coming soon)."
```

Comment:
Are we trying to avoid the "P" word in our documentation now? I don't remember where we ended up on this

Reply:
I think we can continue to use it for now. There isn't a great alternative

Reply:
@0618 Don't you think the meta description should be different for each page though for SEO?

Reply:
I agree! Good point @reesscot  We should use different meta-descriptions on each page. How do we decide what to use on each?

Should I draft a doc for it?

Reply:
doc created https://quip-amazon.com/PPRQA0yE9o5J/Docs-Meta-Descriptions

Reply:
Instead of creating the meta info separately (as in 5ebc04f36110f3ffa09650b45f0da7a74ff26e24), I created a script `createMetaInfo` to auto update them using the meta info in the mdx file, so that we can keep a single source of truth